crates/fj/Cargo.toml

```diff
@@ -10,3 +10,10 @@ repository = "https://github.com/hannobraun/fornjot"
 license = "0BSD"
 keywords = ["cad", "programmatic", "code-cad"]
 categories = ["encoding", "mathematics", "rendering"]
+
+[features]
+default = []
+serialization = ["serde"]
```

Comment:
I think we don't need an explicit feature here. Just having the implicit `serde` feature should be enough, and is what I've seen in other crates in the Rust ecosystem.